common/setups/hybrid/gmm_system.py

```diff
@@ -0,0 +1,878 @@
+__all__ = ["GmmSystem"]
+
+import itertools
+import sys
+from typing import Dict
+
+# -------------------- Sisyphus --------------------
+
+from sisyphus import *
```

Comment:
Such imports of all symbols (`from <...> import *`) are generally not recommended.
But if this is standard code style here, or you think this is fine, then leave it.